packages/runner-docker/scripts/start.sh

```diff
@@ -0,0 +1,128 @@
+#! /bin/bash
+set -e
+
+# Always require the server to run
+if [ -z "$TEST" ]; then
+  echo "\$TEST is required."
+  exit 1;
+fi;
+
+#
+# Write out metadata file
+#
+node metadata.mjs > /home/node/results/metadata.json
+
+# We can mount the path for local dev,
+# If not, require a repo and ref
+REPO_DIR=/home/node/repo
+if [ -d "$REPO_DIR" ]; then
+  echo "Using local benchmarks"
+  cd "$REPO_DIR"
+else
+  if [ -z "$REPO" ]; then
+    echo "\$REPO is required."
+    exit 1;
+  fi;
+
+  if [ -z "$REF" ]; then
+    echo "\$REF is required."
+    exit 1;
+  fi;
+
+  #
+  # Benchmark repo setup
+  #
+  cd $REPO_DIR
+  git init
+  git remote add origin "$REPO"
+  git fetch origin --depth=1 "$REF"
+  git fetch checkout "$REF"
+fi;
+
+
+# A little spinner
```

Comment:
I needed this when testing and running directly. Might be something we can delete since it is explicitly turned off because of the non-interactive shell integration.